src/commissaire_service/storage/__init__.py

```diff
@@ -107,19 +110,19 @@ def register_store_handler(self, config):
         self._manager.register_store_handler(
             handler_type, config, *matched_types)
 
-    def _build_model(self, model_type_name, model_kwargs):
+    def _build_model(self, model_type_name, model_json_data):
```

Comment:
Just for the heck of it up updated to the latest rc of kombu and still get the same issue. However, I can update all the handlers to pass ``json`` rather than ``dict`` which should work. Thoughts?